### ISSUE TITLE
Website is broken and no info in readme?

### ISSUE THREAD

**Issue by FoxxMD** [tomschlick/laravel-micro-manager#5]
The description of this project is enticing but I have no idea how it works or what it does (in detail)! 

The website listed (https://tomschlick.com/laravel-micro-manager/) also does not work.

**Comment by tomschlick** [comment]
Yeah I migrated my site off of github pages so that broke the link. New one is http://tomschlick.github.io/laravel-micro-manager

I still need to finish this package off so it can track the jobs as they are processed through the queue. Basically need to inject and fetch that id to/from the job. I'll try to get that cleared up in the next few days.
